pkg_defs/xija/meta.yaml

```diff
@@ -16,12 +15,10 @@ requirements:
   # Packages required to build the package. python and numpy must be
   # listed explicitly if they are required.
   build:
-    - pip
     - python
-    - setuptools
     - six
     - numba
-    - numpy >=1.12.1
+    - numpy =1.12
```

Comment:
For `shiny` I'm guessing we don't need to pin anything, that the top-level numpy directive specifies the build numpy and ska3-flight specifies the runtime?